include/seqan3/alignment/pairwise/policy/find_optimum_policy.hpp

```diff
@@ -173,20 +173,14 @@ class find_optimum_policy
      *
      * \param[in] cell The current cell to get the score and the coordinate from.
      * \param[in,out] state The state with the current optimum to update.
-     *
-     * \details
-     *
-     * Checks for a new cell in the alignment matrix. If the given score of the alignment matrix cell is greater than
```

Comment:
why did you delete this? 

EDIT: Do you need this function at all now, since you are only calling another function?

Reply:
it does reduce the structured binding in all the other functions. Makes the documentation cleaner and also the code more readable. So I don't mind keeping it.